pro/src/new_components/OfferIndividualForm/UsefulInformations/__specs__/UsefulInformations.spec.tsx

```diff
@@ -93,4 +144,99 @@ describe('OfferIndividual section: UsefulInformations', () => {
     await screen.findByRole('heading', { name: 'Informations pratiques' })
     expect(screen.getByLabelText('Rayonnement national')).toBeInTheDocument()
   })
+
+  it('should contain withdrawal ticket informations when subcategory is an event', async () => {
+    // for an event subCategory
+    initialValues.subcategoryId = 'A-A'
+
+    renderUsefulInformations({
+      initialValues,
+      onSubmit,
+      props,
+    })
+
+    await screen.findByRole('heading', { name: 'Informations pratiques' })
+    expect(
+      await screen.findByText(
+        'Comment les billets, places seront-ils transmis ?'
+      )
+    ).toBeInTheDocument()
+
+    // should contain sent date information when tickets are sent by mail
+    userEvent.click(await screen.findByText('Envoi par e-mail'))
+    expect(await screen.findByText('Date d’envoi')).toBeInTheDocument()
+
+    // should contain withdrawal hour information when tickets are to withdraw on place
+    userEvent.click(
+      await screen.findByText('Retrait sur place (guichet, comptoir ...)')
+    )
+    expect(await screen.findByText('Heure de retrait')).toBeInTheDocument()
+  })
+
+  it('should display an error when the user has not filled all required fields', async () => {
+    // for an event subCategory
+    initialValues.subcategoryId = 'A-A'
+
+    renderUsefulInformations({
+      initialValues,
+      onSubmit,
+      props,
+    })
+
+    expect(
```

Comment:
```suggestion
    renderUsefulInformations({
      initialValues,
      onSubmit,
      props,
    })
    await screen.findByRole('heading', { name: 'Informations pratiques' })

    expect(
```

afin de corrigé un des warning lors de l’exécution du test